webapp/src/app/shared/components/molecules/custom-card/custom-card.component.html

```diff
@@ -1,6 +1,21 @@
 <mat-card class="panel">
     <mat-card-header>
-        {{ card.header }}
+        <span class="title">{{ card.header }}</span>
+        <div *ngIf="tabs">
+            <app-multi-tab-switcher [tabs]="tabs" [tabSelected]="tabSelected" (switchView)="switchTabView($event)"></app-multi-tab-switcher>
+        </div>
+        <app-table-filter-chip
+            *ngIf="filters?.list?.length !== 0 && appliedFiltersDict"
```

Comment:
would be good to extract logic like this to a variable in ts file

Reply:
Done